src/modules/Packages.rb

```diff
@@ -387,12 +387,12 @@ def SummaryOutput(flags)
       end
 
       if Builtins.contains(flags, :pattern)
-        patterns = ListSelected(:pattern, "+  %1")
+        patterns = ListSelected(:pattern, "%1")
 
         if Ops.greater_than(Builtins.size(patterns), 0)
           output = Builtins.add(
             output,
-            Ops.add(_("Patterns:<br>"), Builtins.mergestring(patterns, "<br>"))
```

Comment:
I am not sure whether it's better to remove a line break than keep it translated. Maybe it will look acceptable even with the br?

Reply:
I was unsure too. However, with the `br` looks weird to me (although it is acceptable):

![installer-settings-weird-spacing-detail](https://user-images.githubusercontent.com/15836/39243954-fb2347d6-4886-11e8-87d3-6df650774197.png)

I was considering adding a br after the patterns list, but it looks like a waste of space.